### ISSUE TITLE
api.EventTarget.addEventListener.options - Missing feature for `signal` option

### ISSUE THREAD

**Issue by ilogico** [mdn/browser-compat-data#9555]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener

#### What information was incorrect, unhelpful, or incomplete?
The browser compatibility table.

#### What did you expect to see?
Information about the signal option.

#### Did you test this? If so, how?

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/api/eventtarget/addeventlistener`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/eventtarget/addeventlistener/index.html
* Last commit: https://github.com/mdn/content/commit/f37c24de21200c64921064e536ea591d71df6cb3
* Document last modified: 2021-03-02T22:00:49.000Z

</details>

**Comment by ddbeck** [comment]
@hamishwillee alright, once I had a test case this all came together! Want to do the honors of opening a PR? 😃

OK, I tested this one out, using [this test](https://gist.github.com/ddbeck/5410a42a84789b1fe6942f1b81f03fad) (modified from the example on MDN). The results are:

- Chrome: 90
- Firefox: 86
- Safari: `false`

And we can mirror or presume `false` for the other browsers.

I kinda didn't trust something so new (`addEventListener` is basically primordial, right?) but I was able to find some interesting bugs (thanks to a lucky search against the [gecko mirror](https://github.com/mozilla/gecko-dev)) to back this up:

- Firefox [bug 1679204](https://bugzilla.mozilla.org/show_bug.cgi?id=1679204), shipped in 86
- Chromium [bug 1146467](https://bugs.chromium.org/p/chromium/issues/detail?id=1146467#c5), [shipping in Chrome 90](https://storage.googleapis.com/chromium-find-releases-static/a64.html#a641ee4fd048768763e23a6b4126c1990b64f86b)
- WebKit [bug 218753](https://bugs.webkit.org/show_bug.cgi?id=218753), not yet shipping in Safari
- And the spec change itself: https://github.com/whatwg/dom/issues/911, https://github.com/whatwg/dom/pull/919 ([spec](https://dom.spec.whatwg.org/#ref-for-abortsignal%E2%91%A2))

**Comment by hamishwillee** [comment]
Thanks very much @ddbeck - saved me lots of time, so I guess I will take the BCD task: #9562 :-) 

I get a pass with your test code on Chrome and edge 89. I also got a pass on ,OPera 74 - which corresponds to Chrome 88. My guess would therefore be that this actually went in on Chrome 88 ... but I don't have a version to test that with. I have guessed at chrome 89 just because I can't test. But let's continue that conversation on the PR

**Comment by hamishwillee** [comment]
@chrisdavidmills We seem to have missed this bug in FF86 https://bugzilla.mozilla.org/show_bug.cgi?id=1679204#c4 , which is fixed by a BCD update. Do you want me to add a release note on this? Either way, can you please remove dev-doc-needed because I can't.

**Comment by chrisdavidmills** [comment]
@hamishwillee yes please.

I've updated DDN -> DDC

**Comment by ddbeck** [comment]
I figured out the discrepancy for Chrome 88/90. See https://github.com/mdn/browser-compat-data/pull/9562#discussion_r599401654.

> though it isn't clear how to run a test file like yours on that site

OK, yes, I skipped over that detail. If I have a test locally that I want to try in BrowserStack or Sauce Labs, then I'll use [ngrok](https://ngrok.com/) to serve the file, which creates a publicly-reachable URL. Though sometimes I'll put something into CodePen or the like. It's almost as if I should write some documentation about how to test BCD stuff. 😬
